server/registry/actions_specs_test.go

```diff
@@ -606,7 +606,7 @@ func TestGetApiSpecContentsData(t *testing.T) {
 			if status.Code(err) != test.wantCode {
 				t.Errorf("GetApiSpecContents(%+v) returned status code %q, want %q: %v", test.req, status.Code(err), test.wantCode, err)
 			}
-			if bytes.Compare(resp.Data, test.wantData) != 0 {
+			if !bytes.Equal(resp.Data, test.wantData) {
```

Comment:
This would be easier to discuss as a standalone PR - here I think it adds noise to a PR that already takes some effort to understand, and the questions about this ("is this a styleguide-driven change, a personal preference, or fixing a functional problem?") seem better to discuss separately.

Reply:
This was a lint fix. Yeah, sorry, I'll just stop doing this stuff.